substrate/bin/utils/chain-spec-builder/bin/main.rs

```diff
@@ -17,27 +17,39 @@
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
 use chain_spec_builder::{
-	generate_authority_keys_and_store, generate_chain_spec, print_seeds, ChainSpecBuilder,
+	generate_authority_keys_and_store, generate_chain_spec, generate_chain_spec_for_runtime,
+	print_seeds, ChainSpecBuilder, ChainSpecBuilderCmd, EditCmd, GenerateCmd, NewCmd, VerifyCmd,
 };
 use clap::Parser;
 use node_cli::chain_spec;
 use rand::{distributions::Alphanumeric, rngs::OsRng, Rng};
+use sc_chain_spec::GenericChainSpec;
 use sp_core::{crypto::Ss58Codec, sr25519};
 use std::fs;
 
 fn main() -> Result<(), String> {
+	sp_tracing::try_init_simple();
+
+	let builder = ChainSpecBuilder::parse();
 	#[cfg(build_type = "debug")]
-	println!(
-		"The chain spec builder builds a chain specification that includes a Substrate runtime \
+	if matches!(builder.command, ChainSpecBuilderCmd::Generate(_) | ChainSpecBuilderCmd::New(_)) {
+		println!(
+			"The chain spec builder builds a chain specification that includes a Substrate runtime \
```

Comment:
This message doesn't make any sense anymore as the wasm file should be coming from somewhere else. (aka build manually).

Reply:
will remove it in follow-up PR.